modules/indexing/src/test/java/org/apache/ignite/internal/processors/cache/persistence/db/LongDestroyDurableBackgroundTaskTest.java

```diff
@@ -442,9 +455,18 @@ private void createIndex(IgniteCache<Integer, Integer> cache, boolean multicolum
      * @return Ignite instance.
      * @throws Exception If failed.
      */
-    private IgniteEx prepareAndPopulateCluster(int nodeCnt, boolean multicolumn) throws Exception {
+    private IgniteEx prepareAndPopulateCluster(int nodeCnt, boolean multicolumn, boolean createLsnr) throws Exception {
         IgniteEx ignite = startGrids(nodeCnt);
 
+        if (createLsnr) {
+            GridCacheSharedContext ctx = U.field(ignite.context().cache(), "sharedCtx");
```

Comment:
you can get shared context without using reflection: `ignite.context().cache().context()`